pkgs/top-level/python-packages.nix

```diff
@@ -417,15 +417,16 @@ in {
   };
 
   altair = buildPythonPackage rec {
-    name = "altair-1.0.0";
+    name = "altair-1.2.0";
 
     src = pkgs.fetchurl {
       url = "mirror://pypi/a/altair/${name}.tar.gz";
-      sha256 = "024drhmiw8w3dl7dbal0pvnlfd3sv4n1rqywv2jb488b3fzm704r";
+      sha256 = "05c47dm20p7m0017p2h38il721rxag1q0457dj7whp0k8rc7qd1n";
     };
-
+    # Tests try to execute notebooks that aren't tests:
+    # TypeError: calling <function tutorial at 0x7fffe1bb8048> returned None, not a test
+    doCheck = false;
```

Comment:
figured it out, needs a UTF locale as well, hope this translates to hydra (works locally).